scheduler/src/templates/stripe/success_image.html

```diff
@@ -21,10 +21,11 @@ <h1>Success!</h1>
         </div>
 
         <div class="description">
-            <h2>Using your hotspot</h2>
-            <p>Once the image has been downloaded, flash it onto a microSD card (we recommend <a href="https://www.raspberrypi.com/software/">Raspberry Pi Imager</a>, but any such tool will do).</p>
-            <p>Insert the card into your Raspberry Pi, plug it in, and wait for approximately 1-2 minutes during the initial boot-up process until you see the network status display <em>Ready</em> (or <em>Wikipedia</em> if you purchased Wikipedia image) appear in your wifi settings.</p>
-            <p>Connect to it and a captive portal should appear after a few seconds. If not, then open <a href="http://goto.kiwix.hotspot">http://goto.kiwix.hotspot</a> in your favourite browser.</p>
+            <h2>Downloading and your hotspot</h2>
+            <p>Bot links work, but torrent will restart where it stopped in case your connection breaks down and it will also check for file corruption after completion. <a href="https://www.transmissionbt.com">Transmission</a> is the client that we recommend.</p>
```

Comment:
Ah “both”, I thought it was slang I didn't knew about 🤦‍♂️

OK please suggest another sentence here that highlights the torrent. I'll push the change with the typo fix.

Reply:
How about `Torrent-based downloads are more resilient and will for instance restart where they stopped in case your connection breaks down. They will also check for file corruption after completion. We recommend using <a href="https://www.transmissionbt.com">Transmission</a>.</p>